docs/installation.rst

```diff
@@ -6,7 +6,6 @@ Installation
 We try to follow `SPEC0 <https://scientific-python.org/specs/spec-0000/>`_ as far as minimum supported dependencies, with the following caveats:
 
 - Python 3.10, 3.11, 3.12 - **we do not yet support Python 3.13**
-- OpenMM 8.0, 8.1.1, 8.1.2 - **we do not yet support OpenMM v8.2.0**
```

Comment:
I would keep "we do not support OpenMM v8.3.0"

Reply:
ah good point